pkg/action/executor/k8s_controller_updater.go

```diff
@@ -47,20 +49,34 @@ func newK8sControllerUpdaterViaPod(clusterScraper *cluster.ClusterScraper, pod *
 	if discoveryutil.IsOwnerInfoEmpty(ownerInfo) {
 		return nil, fmt.Errorf("pod %s/%s does not have controller", pod.Namespace, pod.Name)
 	}
-	return newK8sControllerUpdater(clusterScraper, ormClient, ownerInfo.Kind, ownerInfo.Name, pod.Name, pod.Namespace)
+	// TODO: For an action on a parent workload controller of this pod managed by gitops (argoCD) we will need
+	// Some information to be available on the entity we get in the action (pod here).
```

Comment:
Nit: The starting "S" should be lower case.

Reply:
Updated!